packages/react/src/components/Modal/Modal.mdx

```diff
@@ -1,13 +1,295 @@
-import { Props } from '@storybook/addon-docs/blocks';
+import { Story, Props } from '@storybook/addon-docs/blocks';
+import Modal from '../Modal';
 
 # Modal
 
```

Comment:
Can we generate a Table of Contents here, as well as add in the external link section? 

```suggestion
[Source code](https://github.com/carbon-design-system/carbon/tree/master/packages/react/src/components/Modal)
&nbsp;|&nbsp;
[Usage guidelines](https://www.carbondesignsystem.com/components/modal/usage)
&nbsp;|&nbsp;
[Accessibility](https://www.carbondesignsystem.com/components/modal/accessibility)


## Table of Contents

- [Overview](#overview)
- [Component API](#component-api)
- [References](#references)
- [Feedback](#feedback)
```

Reply:
how is this being generated? doctoc or something?

Reply:
Yeah I used `doctoc` but you can just copy and paste a general Table of Contents 👍